ui/pages/confirmations/components/confirm/footer/footer.tsx

```diff
@@ -61,8 +78,17 @@ const ConfirmButton = ({
   const [confirmModalVisible, setConfirmModalVisible] =
     useState<boolean>(false);
 
-  const { dangerAlerts, hasDangerAlerts, hasUnconfirmedDangerAlerts } =
-    useAlerts(alertOwnerId);
+  const {
+    hasDangerAlerts,
+    hasUnconfirmedDangerAlerts,
+    fieldAlerts,
+    isAlertConfirmed,
+  } = useAlerts(alertOwnerId);
+
+  const unconfirmedDangerAlertsFields = fieldAlerts.filter(
```

Comment:
Same question here, do we want to also `Review alerts` if there is more than one general danger alert since that would mean we show the aggregate message in the confirmation instead?

Reply:
Same as [above](https://github.com/MetaMask/metamask-extension/pull/27205#discussion_r1768606292). We may need to revisit this in DSU because it means we'll be back to stacking banner alerts.